public/build_config/webpack.prod.conf.js

```diff
@@ -1,55 +1,68 @@
 var webpack = require('webpack');
 var path = require('path');
-var ExtractTextPlugin = require('extract-text-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 module.exports = {
   bail: true,
+  mode: 'production',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'app.js'
+  },
   module: {
-    loaders: [
+    rules: [
       {
-        test:    /\.js$/,
+        test: /\.js$/,
+        use: [{
+          loader: "babel-loader",
+          options: {
+            presets: ["@babel/preset-env", "@babel/preset-react"]
+          }
+        }],
```

Comment:
are there enough overlaps in the webpack conf files to create a common config file and use webpack-merge? maybe one for another pr (upgrading to webpack v5?)